clients/python/sust/api/__init__.py

```diff
@@ -1,5 +0,0 @@
-from .climate_explorer import (
```

Comment:
Add this file back in. I haven't excluded it from the area of effect from the generation process yet

Reply:
Yep, that was removed by the OpenAPI generator.